### PR TITLE
For bulk data response, there is no need to parse, toString, and then parse the json again

### PR DESCRIPTION
**Pull request by dcrissman**

### CODE REVIEW

File: core/src/main/java/com/redhat/lightblue/client/response/AbstractLightblueResponse.java [hunk 1]
```diff
@@ -30,6 +30,16 @@ public AbstractLightblueResponse(String responseText, ObjectMapper mapper) throw
         }
     }
 
+    public AbstractLightblueResponse(JsonNode responseNode) {
+        this(responseNode, JSON.getDefaultObjectMapper());
+    }
+
+    public AbstractLightblueResponse(JsonNode responseNode, ObjectMapper mapper) {
```
Comment (bserdar) [inline core/src/main/java/com/redhat/lightblue/client/response/AbstractLightblueResponse.java]: Is there any point passing this mapper around?

Comment (dcrissman) [inline core/src/main/java/com/redhat/lightblue/client/response/AbstractLightblueResponse.java]: I would love to say no, but DefaultLightblueDataResponse#parseProcessed uses it, so it needs to be there. The intention is that the default will always be used, unless a custom one is provided, in which case the custom one will always be used.

**Comment by bserdar** [comment]
It says conflicts with base, needs rebase

**Comment by dcrissman** [comment]
@bserdar base merged in

**Pull request merged**